src/js/shield_defs.js

```diff
@@ -4011,6 +4011,16 @@ export function loadShields() {
         Color.shields.white
       ))
   );
+  shields["NL:DR:Hunebed_Highway"] = {
+    spriteBlank: ["shield_nl_dr_hunebed"],
+    textColor: Color.shields.white,
+    padding: {
+      left: 4,
+      right: 4,
+      top: 6,
+      bottom: 4,
+    },
```

Comment:
Do we need to decrease the padding or make the overall shield larger in both dimensions? The original image was designed for only two characters, so stuffing three characters in there could make it difficult to discern the number.

Reply:
> Or does the concurrency with the usual N34 shield makes discerning the number much less important?

Basically this, but I've now widened the shield anyway.